docs/getting-started-guides/centos/centos_manual_config.md

```diff
@@ -1,236 +0,0 @@
----
-assignees:
-- thockin
-title: CentOS
----
-
```

Comment:
While I would normally agree with that I have been working with new users for the past few weeks and the sheer amount of content and old/broken/miss information is causing more confusion than I think it's worth to keep an old version of a document around and redirect to where someone should be looking. This makes google searching and searching within the documentation itself more confusing as users are less likely to read a disclaimer if the page exists and shows up at the top of their search.
I think having CentOS documentation is valuable to have but this document needs a complete re-write to be up to date with best practices and get an up to date cluster running (including kube-proxy, dns, addons). I also think a lot of that effort is being spent on kubeadm so ideally people would find that documentation before anything else.